lib/opam.ml

```diff
@@ -189,24 +170,6 @@ module Package_summary = struct
     depexts : (OpamSysPkg.Set.t * OpamTypes.filter) list;
   }
 
-  let equal { package; url_src; hashes; dev_repo; depexts } t' =
-    OpamPackage.equal package t'.package
-    && Option.equal Url.equal url_src t'.url_src
-    && List.equal Hash.equal hashes t'.hashes
-    && Option.equal String.equal dev_repo t'.dev_repo
-    && Depexts.equal depexts t'.depexts
-
-  let pp fmt { package; url_src; hashes; dev_repo; depexts } =
```

Comment:
I would keep `pp` because… I often use it for print debugging and it is convenient to be able to do a debug print every now and then :see_no_evil:

Reply:
Fair enough. I've added back `pp`. Looks like implementing `pp` for build commands won't be too hard.